package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "node-ember-cli-deploy-redis",
-  "version": "0.0.1",
-  "description": "A node middleware to serve EmberJS apps deployed by ember-cli-deploy",
+  "version": "1.0.0",
```

Comment:
I'd suggest keeping this below 1 until you have more adoption and feedback and feel confident in the API stability.